docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
```

Comment:
Should `Data View` always be capitalized here?

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/171725316-7e8436a3-4ef9-4d24-92b2-20f78dcbaa56.png" />
</p>

Reply:
Just when it's referring to the button I believe, since it should copy the capitalization of the button.

Reply:
Ahh, yeah -- makes sense! Also seems the [main docs reference](https://www.elastic.co/guide/en/kibana/master/data-views.html) as lowercase.